blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -0,0 +1,344 @@
+  }, [configuredOffset, configuredSize, contentService]);
+
+  const requestedResultList = useContent({
+    source: isServerSideLazy ? null : contentService,
+    query: {
+      ...contentConfigValues,
+      feature: 'results-list',
+      ...serviceQueryPage(queryOffset),
+    },
+    filter: `{
+      count
+      next
+      content_elements {
+        _id,
+        type
+        display_date
+        credits {
+          by {
+            _id
+            name
+            url
+            type
+            additional_properties {
+              original {
+                byline
+              }
+            }
+          }
+        }
+        headlines {
+          basic
+        }
+        description {
+          basic
+        }
+        promo_items {
+          basic {
+            type
+            url
+            resized_params {
+              274x154
+              158x89
+            }
+          }
+          lead_art {
+            promo_items {
+              basic {
+                type
+                url
+                resized_params {
+                  274x154
+                  158x89
+                }
+              }
+            }
+            type
+          }
+        }
+        websites {
+          ${arcSite} {
+            website_url
+          }
+        }
+      }
+    }`,
+  });
+
+  const [resultList, alterResultList] = useReducer(reduceResultList, requestedResultList);
+
+  useEffect(() => {
+    if (requestedResultList) {
+      alterResultList({
+        type: 'appendUnique',
+        data: requestedResultList,
+      });
+    }
+  }, [requestedResultList]);
+
+  useEffect(() => {
+    if (queryOffset !== configuredOffset) {
+      const topItem = resultList.content_elements[queryOffset - configuredOffset];
+      if (topItem?._id && listItemRefs.current[topItem?._id]) {
+        const focusLink = listItemRefs.current[topItem._id].querySelector('a:not([aria-hidden])');
+        if (focusLink) {
+          focusLink.focus();
+        }
+      }
+    }
+  }, [configuredOffset, queryOffset, resultList]);
+
+  const [promoElements] = useState(resolveDefaultPromoElements(customFields));
+  const [phrases] = useState(getTranslatedPhrases(locale || 'en'));
+
+  const viewableElements = resultList?.content_elements
+    .slice(0, queryOffset + configuredSize - configuredOffset);
+
+  return (viewableElements?.length > 0 && !isServerSideLazy) ? (
+    <div className="results-list-container">
+      {viewableElements.map((element) => (
+        <ResultsItem
+          key={`result-card-${element._id}`}
+          ref={listItemRefs.current[element._id]}
+          arcSite={arcSite}
+          element={element}
+          promoElements={promoElements}
+          imageProps={imageProps}
+          targetFallbackImage={targetFallbackImage}
+          placeholderResizedImageOptions={placeholderResizedImageOptions}
+        />
+      ))}
+      {(viewableElements.length < resultList?.content_elements.length
```

Comment:
See more button is showing even when the results list is shorter than the custom field size value.
Example 
content source - story-feed-query 
quey - matthew
size - 50

There are only 2 results for query `matthew` - but See More button is still shown

Reply:
Resolved in 5286b9a